index.html

```diff
@@ -51,7 +51,115 @@ <h3>I'm a Software Developer</h3>
 
-</div>
+<section class="page2">
+    <div class="container">
+        <div class="items title1">
+           <h3>My Recent Works</h3>
+        <br> <br>
+        <hr>
+        </div>
+        <div class="items img1">
+            <img src="images/testimg.png" alt="">
+        </div>
+        <div class="items description1">
+            <h3 class="title2">Multi-Post Stories</h3>
+            <p class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
+        <ul class="items lang">
+            <li>html</li>
+            <li>css</li>
+            <li>php</li>
+            <li>mysql</li>
+            <li>javascript</li>
+            
+        </ul>
+        <div class="items buttontypeA">
+        <a href="#">See Project</a>
+        </div>
+        <div class="items site1">
+            <div class="description2">
+                <div class="title3">Multi-Post Stories</div>
+                <p class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            </div>
+            <ul class="items lang">
+                <li>html</li>
+                <li>css</li>
+                <li>php</li>
+                <li>mysql</li>
+                <li>javascript</li>
+                
+            </ul>
+            <div class="items buttontypeB">
+                <a href="#">See Project</a>
+            </div>
+        </div><!--site1 end-->
+        <br> <br> <br>
+        <div class="items site1 site2">
+            <div class="description2">
+                <div class="title3">Multi-Post Stories</div>
+                <p class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            </div>
+            <ul class="items lang">
+                <li>html</li>
+                <li>css</li>
+                <li>php</li>
+                <li>mysql</li>
+                <li>javascript</li>
+                
+            </ul>
+            <div class="items buttontypeB">
+                <a href="#">See Project</a>
+                </div>
+        </div><!--site2 end-->
+        
+    </div><!--container end-->
+</section><!--page2 end-->
+<br> <br> <br>
+<section class="page3">
+    <div class="container">
+        <br> <br> 
+        <h3 class="items title1">About me</h3>
+        <p class="items text1">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
```

Comment:
- [OPTIONAL] kindly align your `h3` tag so that it does not overlap with your `p` tag
  Your design 👇 
              
![image](https://user-images.githubusercontent.com/66097371/148509463-c0e72fcc-2d58-4194-99f7-136cdbe230db.png)
Figma design 👇 
        
![image](https://user-images.githubusercontent.com/66097371/148509545-61e8ebf8-099d-42b1-ada4-33d7054f1264.png)